module/VuFind/src/VuFind/Log/Handler/MailHandler.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace VuFind\Log\Handler;
+
+use Monolog\Handler\MailHandler as MonologMailHandler;
+use Monolog\LogRecord;
+use Monolog\Logger;
+use VuFind\Log\Handler\VerbosityTrait;
+use VuFind\Mailer\Mailer;
+
+/**
+ * Custom Mail Handler for VuFind with verbosity support and VuFind mailer integration
+ */
+class MailHandler extends MonologMailHandler
+{
+    use VerbosityTrait;
+
+    /**
+     * @var string The recipient email address
+     */
+    protected $to;
+
+    /**
+     * @var string The sender email address  
+     */
+    protected $from;
+
+    /**
+     * @var string The email subject
+     */
+    protected $subject;
+
+    /**
+     * @var Mailer VuFind's mailer instance
+     */
+    protected $mailer;
+
+    /**
+     * Constructor
+     *
+     * @param string $to      Recipient email address
+     * @param string $subject Email subject
+     * @param string $from    Sender email address
+     * @param Mailer $mailer  VuFind mailer instance
+     */
+    public function __construct(string $to, string $subject, string $from, Mailer $mailer)
```

Comment:
You could simplify this by removing the property definitions above and assignments below, and just using:

```suggestion
    public function __construct(protected string $to, protected string $subject, protected string $from, protected Mailer $mailer)
```